shopify-client/src/views/ShopList.vue

```diff
@@ -0,0 +1,261 @@
+<template>
+  <div class="shopList">
+    <h1>Shop List</h1> <!--Temp header-->
+    <div id="filters">
+      <div class="filterTile">
+        <h4>Categories</h4>
+        <div v-if="categories.length < 1">
+          <p>No available categories</p>
+        </div>
+        <div
+            v-else
+            v-for="category in categories"
+            :key="category.id"
+        >
+          <label :for="category.name">{{category.name}}</label>
+          <input
+              type="checkbox"
+              :id="category.name"
+              class="checkBox"
+          >
+        </div>
+      </div>
+      <div class="filterTile">
+        <h4>Tags</h4>
+        <div v-if="tags.length < 1">
+          <p>No available tags</p>
+        </div>
+        <div
+            v-else
+            v-for="tag in tags"
+            :key="tag.id"
+        >
+          <label :for="tag.name">{{tag.name}}</label>
+          <input
+              type="checkbox"
+              :id="tag.name"
+              class="checkBox"
+          >
+        </div>
+      </div>
+      <input
+          id="filterButton"
+          type="button"
+          value="Filter Search"
+          v-on:click="filterSearch"
+      />
+    </div>
+    <div id="filteredList">
+      <h4 id="searchKeyword">Search keyword: "{{keyword}}"</h4>
+      <div v-if="shops.length < 1">
+        No results found
+      </div>
+      <div
+          v-else
+          v-for="shop in shops"
+          :key="shop.id"
+          class="shopTile"
+      >
```

Comment:
Use the bootstrap card component instead of making your own. https://bootstrap-vue.org/docs/components/card.